core/tasks/yfinance_sync.py

```
@@ -0,0 +1,87 @@
+            "results": results,
+            "status": "completed",
+        }
+
+    except Exception as e:
+        logger.exception(f"Error syncing {stock_code}: {e}")
+        raise self.retry(exc=e)
+
+
+@shared_task(bind=True, max_retries=1)
+def sync_multiple_stocks_task(
+    self,
```

Ruff (0.14.11)

Unused function argument: `self`

(ARG001)